spec/cc/engine/analyzers/javascript/main_spec.rb

```diff
@@ -124,6 +124,14 @@
   def engine_conf
     CC::Engine::Analyzers::EngineConfig.new({
       'config' => {
+        'checks' => {
+          'similar-code' => {
+            'enabled' => true,
+          },
+          'identical-code' => {
+            'enabled' => true,
```

Comment:
this change isn't needed for the specs to pass, since the checks are turned on by default, but there may be some value in having the example.